RELEASING.md

```diff
@@ -1,25 +1,30 @@
 # Release Steps
 
 To release artifacts 'dependencies-parent', 'dependencies', and 'linkage-checker-enforcer-rules',
-follow the ["Developers/Releasing" steps in go/ct4j](
-https://g3doc.corp.google.com/company/teams/cloud-java/tools/developers/releasing.md?cl=head).
+run `prepare_release.sh` with `dependencies` argument:
 
-Note: Cloud OSS BOM release procedure [boms/cloud-oss-bom/RELEASING.md](
+```
+$ cd cloud-opensource-java
+$ git checkout master
+$ git pull
+$ ./script/prepare_release.sh dependencies 1.0.0
+```
+
+Note: Google Cloud Libraries BOM release procedure [boms/cloud-oss-bom/RELEASING.md](
 boms/cloud-oss-bom/RELEASING.md) is not part of this document.
 
+Create a PR for the release, and get it approved.
+
 ## Rapid builds
 
+Run the following Rapid release pipelines:
+
 | Artifact ID | Rapid project | Dependency |
 | ---------- | ------------------ | --------- |
 |dependencies-parent| [cloud-java-tools-cloud-opensource-java-parent-kokoro-release](http://rapid/cloud-java-tools-cloud-opensource-java-parent-kokoro-release)||
 |dependencies| [cloud-java-tools-cloud-opensource-java-dependencies-kokoro-release](http://rapid/cloud-java-tools-cloud-opensource-java-dependencies-kokoro-release)|dependencies-parent|
 |linkage-checker-enforcer-rules|[cloud-java-tools-cloud-opensource-java-enforcer-rules-kokoro-release](http://rapid/cloud-java-tools-cloud-opensource-java-enforcer-rules-kokoro-release)|dependencies-parent, dependencies|
 
-When releasing a version of `linkage-checker-enforcer-rules`, you need to release `dependencies`
```

Comment:
This sentences were unnecessarily unclear. As of now we release them altogether; there's no case where we only release `dependencies`.